### PR TITLE
remove desword from nukie uplink

### PR DESCRIPTION
**Pull request by Emisse**
the nightmares wont stop

🆑 
- remove: Removed desword from nukie uplink.

### CODE REVIEW

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 1]
```diff
@@ -95,7 +95,12 @@
     Telecrystal: 16
   categories:
   - UplinkWeapons
-
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:100]: Change this to "blacklist" if you want it to me removed from nukeops

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 1 above)
Comment (Emisse) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:100]: so true

**Comment by KaiShibaa** [comment]
Considering from what I've seen I agree with the removal. 

A team of nukies with deswords sounds insane considering the reflection and pseudo-resistances.

**Comment by Emisse** [comment]
glowing approval no objections send it

**Pull request merged**